.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+Resolves #ISSUE.
+
+**No (!) PRs should be merged without at least one review from a different team member of the Mondo Technical team.**
+
+## Documentation
```

Comment:
An alternative to checkbox strategy. Rather than (a) a bunch of sections where the user selects 1 of several options, we could (b) have a "pre-merge checklist" where we make sure every box is ticked.

```suggestion
## Pre-merge checklist
- [ ] Docs
a. `docs/` have been added/updated **OR**
b. No updates to the docs is considered unnecessary after careful consideration.
- [ ] QC
a. `sh run.sh make build-mondo-ingest` has been run on this branch (after `docker pull obolibrary/odkfull:dev), and no errors occurred **OR**
b.  No functional (code-related) changes to the pipeline are suggested, so no re-run is necessary.
- [ ] Reviewed
Has been sufficiently reviewed by at least one review from a different team member of the Mondo Technical team.
```

### Possible additional sections
- **Reviewed**: The "Reviewed" sub-task could be checked off by one of the reviewers.
- **High level summary description**
- **Referenced an issue which this PR addresses**

### Before/after
<details><summary>Before</summary>
<p>

**No (!) PRs should be merged without at least one review from a different team member of the Mondo Technical team.**

## Documentation
- [x] `docs/` have been added/updated **OR**
- [ ] No updates to the docs is considered unnecessary after careful consideration.

## QC
- [ ] `sh run.sh make build-mondo-ingest` has been run on this branch (after `docker pull obolibrary/odkfull:dev), and no errors occurred **OR**
- [x] No functional (code-related) changes to the pipeline are suggested, so no re-run is necessary.

</p>
</details> 

<details><summary>After</summary>
<p>

## Pre-merge checklist
- [ ] Docs
a. `docs/` have been added/updated **OR**
b. No updates to the docs is considered unnecessary after careful consideration.
- [ ] QC
a. `sh run.sh make build-mondo-ingest` has been run on this branch (after `docker pull obolibrary/odkfull:dev), and no errors occurred **OR**
b.  No functional (code-related) changes to the pipeline are suggested, so no re-run is necessary.
- [ ] Reviewed
_Has been sufficiently reviewed by at least one review from a different team member of the Mondo Technical team._
</p>
</details>